### PR TITLE
Fix compilation of the preview in editing forms

### PR DESCRIPTION
**Pull request by asaunier**
Fix
> not sure angular code is correctly executed within the modal preview (for instance it seems that the outing dates are not correctly shown with angular moment and that the interface strings translations does not work)

see https://github.com/c2corg/v6_ui/issues/168#issuecomment-245893257

=> translations and date are now displayed as expected

### CODE REVIEW

File: c2corg_ui/static/js/documentediting.js [hunk 1]
```diff
@@ -559,12 +548,24 @@ app.DocumentEditingController.prototype.preview = function() {
...
   };
+  // TODO: loading indicator?
   this.http_.post(url, payload, config).
     catch(function(response) {
       this.alerts.addError('A server error prevented creating the preview');
     }.bind(this)).
     then(function(response) {
-      $('#preview-container-content').html(response['data']);
+      var template = angular.element('#preview-container').clone();
+      template.contents().filter(function() {
```
Comment (tsauerwein) [inline c2corg_ui/static/js/documentediting.js]: You want to get the element with id `preview-container-content` inside the element `#preview-container`? Maybe you could use [find](https://api.jquery.com/find/) instead?

Comment (asaunier) [inline c2corg_ui/static/js/documentediting.js]: I know :)
But the angular doc https://docs.angularjs.org/api/ng/function/angular.element says that 
> find() - Limited to lookups by tag name

**Comment by asaunier** [comment]
Thanks for reviewing @tsauerwein I will merge when the articles are merged, this way I will apply the preview change to them as well.

> Great that you found a fix.

Yes, now I have no excuse for not implementing the original design in the user profiles https://github.com/c2corg/v6_ui/issues/335 :P

File: c2corg_ui/static/js/documentediting.js (hunk 1 above)
Comment (asaunier) [inline c2corg_ui/static/js/documentediting.js]: OK, then we can use ``find``. I have updated the PR (also with the change in articles).
I will merge when Travis passes then.

**Pull request merged**